### PR TITLE
chore(HACBS-1505): Update FBC pipeline with new IIB endpoint

### PR DESCRIPTION
**Pull request by theflockers**
Updates the iib pipeline template adding the proper endpoints and parameters according to the changes done by release-engineering to implement the `fbc-operations` endpoint.

- reorders the parameters
- changes the endpoints
- updates the base image

### CODE REVIEW

**Comment by theflockers** [comment]
@scoheb the pipeline should have the name `iib` as the operator needs it to create the new pipelinerun.

File: openshift/iib-template.yaml [hunk 1]
```diff
@@ -23,9 +23,11 @@ objects:
         - name: iibServiceConfigSecret
           type: string
           description: Secret containing IIB service Config
+          default: iib-production-services-config
```
Comment (scoheb) [inline openshift/iib-template.yaml:26]: let's change this to `iib-services-config`

Comment (scoheb) [reply openshift/iib-template.yaml:26]: we'll inject the correct secret depending on the environment.

**Review by scoheb — CHANGES_REQUESTED** [review]
comments added

**Review by scoheb — APPROVED** [review]

**Pull request merged**